### ISSUE TITLE
[Python] Implement new Concept Exercise: exiting loops

### ISSUE THREAD

**Issue by DavidGerva** [exercism/v3#1432]
This issue describes how to implement the exiting loops with `break` and `continue` statements concept exercise for the Python track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

This concept exercise should convey a basic understanding of the `break` and `continue` statements in Python to interrupt or change the normal flow of a `while` or `for` loop.

## Learning objectives

- use the `break` statement to terminate the nearest enclosing loop
- use the `continue` statement to continue with the next cycle of the nearest enclosing loop

## Out of scope

- nothing

## Concepts

- `break` statement
- `continue` statement

## Prerequisites

- `while` loop
- `for` loop

## Resources to refer to

- [The break statement](https://yawpitchroll.com/posts/the-35-words-you-need-to-python/#break)
- [The break statement python docs](https://docs.python.org/3/reference/simple_stmts.html#the-break-statement)
- [The continue statement](https://yawpitchroll.com/posts/the-35-words-you-need-to-python/#continue)
- [The continue statement python docs](https://docs.python.org/3/reference/simple_stmts.html#the-continue-statement)

### Hints

- TBD?

### After

Nothing

## Representer

No changes required.

## Analyzer

No changes required.

## Implementing

Tests should be written using unittest.TestCase and the test file named bool_basic_test.py.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

## Edits
-

**Comment by aimorris** [comment]
This and #1430 should be merged to one concept exercise `loops`, where you can also explain the usage of `for` loops.

**Comment by DavidGerva** [comment]
Merged into #1430
